cmd/config/runner/runner.go

```diff
@@ -74,11 +76,11 @@ func (e ExecuteCmdOnPkgs) Execute() error {
 func (e ExecuteCmdOnPkgs) processPkg(pkgPath string) error {
 	// Add schema present in openAPI file for current package
 	if e.NeedOpenAPI {
-		clean, err := openapi.AddSchemaFromFile(filepath.Join(pkgPath, ext.KRMFileName()))
+		sc, err := openapi.SchemaFromFile(filepath.Join(pkgPath, ext.KRMFileName()))
 		if err != nil {
 			return err
 		}
-		defer clean()
+		e.SetterSchema = sc
```

Comment:
Agree. I missed renaming in some areas. Renamed to `SettersSchema`

We need not get the schema here. Removed this section in `runner`. For each package, new CmdRunner will be initialized with new schema rather than reassigning it. [Example here](https://github.com/kubernetes-sigs/kustomize/pull/3219/files#diff-8ec671d9c3737641b4c08ae6c47a5c1a0913409f3d78763672223dc905feb92aR174)